Sming/Arch/Esp8266/options.json

```diff
@@ -0,0 +1,10 @@
+{
+    "vdd": {
+        "description": "Modify PHY configuration to support `system_get_vdd33()` function",
+        "partitions": {
+            "phy_init": {
+                "filename": "$(FLASH_INIT_DATA_VCC)"
+            }
+        }
+    }
+}
```

Comment:
@slaff Here's the glue for the 'vdd' option, just needs your code to implement `FLASH_INIT_DATA_VCC`